examples/version_update_single_node.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Update a HyperCore single-node system
+  hosts: localhost
+  connection: local
+  gather_facts: false
+
```

Comment:
I will add 'check: true' to example. If someone runs example, but forgets to change SC_HOST first - in such case it is safer to not actually start update.

Reply:
Agreed and thank you.